todo_app/app.py

```diff
@@ -1,40 +1,51 @@
 from flask import Flask, render_template, request
 
-from todo_app.flask_config import Config
-import requests 
+from todo_app.flask_config import Config # when done with new def can be removed? 
```

Comment:
I'm not certain what this comment means; you currently use the Config on line 13 so removing the import alone will fail, but if you're calling out that you probably don't need the secret key set at all after moving to Trello you might well be right!